samples/bluetooth/mesh/light/README.rst

```diff
@@ -92,8 +100,10 @@ Source file setup
 
 This sample is split into the following source files:
 
-* A :file:`main.c` file to handle initialization.
-* One additional file for handling mesh models, :file:`model_handler.c`.
+* :file:`main.c` - used to handle initialization.
+* :file:`model_handler.c` - used to handle mesh models.
+* :file:`thingy53.c` - Compiled only when the sample is build for :ref:`zephyr:thingy53_nrf5340` board.
+  Used to handle preinitialization of the :ref:`zephyr:thingy53_nrf5340` board.
```

Comment:
```suggestion
* :file:`main.c` used to handle initialization.
* :file:`model_handler.c` used to handle mesh models.
* :file:`thingy53.c` used to handle preinitialization of the :ref:`zephyr:thingy53_nrf5340` board.
  Only compiled when the sample is built for the :ref:`zephyr:thingy53_nrf5340` board.
```
The capitalization was inconsistent, and the hyphens weren't really needed. 
Changing the order of the two sentences for `thingy53.c` makes that bullet point more consistent with the other two.

Reply:
Did same for the other sample too.